backend/api/swagger/pipeline.swagger.json

```diff
@@ -150,6 +156,7 @@
         ]
       },
       "delete": {
+        "summary": "Delete a pipeline version by pipeline version ID. If the deleted pipeline\nversion happens to be the default pipeline version of a certain pipeline,\nthe default version of that pipeline will change to the latest pipeline\nversion among all the remaining pipeline versions of that pipeline. If\nthere is no remaining pipeline version, that pipeline will have no default\nversion. Examine the run_service_api.ipynb notebook to [learn more about creating a run using a pipeline version](https://github.com/kubeflow/pipelines/blob/master/tools/benchmarks/run_service_api.ipynb)",
```

Comment:
Suggested Revision:

`Deletes a pipeline version by pipeline version ID. If the deleted pipeline version is the pipeline's default pipeline version, the default version of that pipeline changes to the pipeline's most recent remaining pipeline version. If there are no remaining pipeline versions, that pipeline will not have a default version. Examine the run_service_api.ipynb notebook to [learn more about creating a run using a pipeline version](https://github.com/kubeflow/pipelines/blob/master/tools/benchmarks/run_service_api.ipynb)`

Reply:
addressed in proto file